### DISCUSSION TITLE
Zô thảo luận nè các bạn iu

### DISCUSSION THREAD

**Discussion by GiaHan06** [GiaHan06/Lop-10A02-Nhom-02#11] [General]
Tham khảo chỗ nào nhớ gắn link zô nheee

**Comment by HuynhLeBaoTran** [comment]
Ckào mấy ghệ iuuuuuu

**Reply by GiaHan06** [reply]
hế lô

**Comment by HuynhLeBaoTran** [comment]
Dô nè dô nè

**Comment by GiaHan06** [comment]
Ulatr biết gì trưa

**Comment by HuynhLeBaoTran** [comment]
j dọ j dọ

**Comment by LTGHan** [comment]
J dọ tui hôk coi đc

**Reply by GiaHan06** [reply]
ủa hong được thiệt

**Comment by chaudambaotram** [comment]
hế lu mọi người

**Comment by NguyenNgocQuynhAnh** [comment]
ủa gòy giờ làm chưa za

**Comment by GiaHan06** [comment]
[Câu hỏi tìm hiểu Tuần 10.docx](https://github.com/GiaHan06/Lop-10A02-Nhom-02/files/7503525/Cau.h.i.tim.hi.u.Tu.n.10.docx)
ũa jztr :((

**Reply by chaudambaotram** [reply]
chưa xong bài 5 và bài 6 nữa

**Reply by GiaHan06** [reply]
chắc xĩu ngang

**Reply by chaudambaotram** [reply]
có luôn bài 7 và bài 8

**Comment by HuynhLeBaoTran** [comment]
ũa j za

**Comment by GiaHan06** [comment]
thui làm đi nè mấy bà ơi, có gì hong hiểu lắm thì zoo đây

**Comment by HuynhLeBaoTran** [comment]
okk

**Comment by LTGHan** [comment]
Shocku

**Comment by GiaHan06** [comment]
Tham khảo chỗ nào nhớ gắn link zô nữa nhe mấy bà oii

**Reply by chaudambaotram** [reply]
mk tham khảo bao nhiu web thì đều gắn vào hết lun hẻ?

**Reply by GiaHan06** [reply]
:)) ủa tui hong biết nữa, nếu mà mấy cái nội dung trùng nhau thì lấy 1 cái nào đó thui, hoặc lấy mấy cái mà bà tham khảo chính á

**Reply by chaudambaotram** [reply]
okokok! chứ tui gắn vào hết chắc cỡ 3 web

**Reply by GiaHan06** [reply]
ok lun nè

**Comment by GiaHan06** [comment]
ũa sao tui hong thả hình zô được ta :(( thấy mấy cái trên canva đẹp lắm mà hong biết chọn cái nào :((

**Comment by LTGHan** [comment]
:')))

**Comment by GiaHan06** [comment]
mấy bà ơiiii

**Reply by GiaHan06** [reply]
https://www.canva.com/design/DAEvObyYB70/share/preview?token=JAlPtn_KGTrT_dnI8jPYBA&role=EDITOR&utm_content=DAEvObyYB70&utm_campaign=designshare&utm_medium=link&utm_source=sharebutton
tui có kím được 3 mẫu xinh xĩu luôn nè:)) mà do cái này là nằm ngang á nên đổi kích cỡ hong biết nó có còn được giống v hong :< zô xem thử ik

**Reply by GiaHan06** [reply]
mỗi mẫu tui chỉ để có 3 trang thui nha

**Reply by GiaHan06** [reply]
nếu mà vô link đấy mấy bà thấy mấy cái mẫu ở phía bên trái á thì thích cái nào lựa luôn nhee:3 mình chọn ra 2 cái lận á, hong ấy chọn luôn 4 mẫu để dành cho tuần 10

**Comment by GiaHan06** [comment]
@LTGHan mỗi đứa 1 bài toán nhe, còn bài toán còn lại đứa liệt kê bước đứa sơ đồ khối nha

**Reply by LTGHan** [reply]
Ok

**Reply by LTGHan** [reply]
Mà bài cuối tui lm sơ đồ khối ha

**Comment by LTGHan** [comment]
Hm

**Reply by GiaHan06** [reply]
ê câu 4 có 2 ý lận á nha
